### ISSUE TITLE
Will Atlantis be supported in Proxyman for Windows?

### ISSUE THREAD

**Issue by radianttap** [ProxymanApp/atlantis#118]
As the title says, this is just inquiry is it planned / possible on Windows?

**Comment by NghiaTranUIT** [comment]
Thanks for the suggestion. Luckly, the Bounjour Service can be implemented on the Windows build. Thus, it's feasible.

I will keep this ticket open and soon work on it 👍

**Comment by radianttap** [comment]
Fantastic! I think I already know who will be first customers for it ;)

**Comment by Banck** [comment]
Hey!
Is there any progress? :-)

**Comment by NghiaTranUIT** [comment]
Just wondering: Would you like to use iOS + Atlantis, and send the traffic to Proxyman for Windows?

**Comment by Banck** [comment]
> Just wondering: Would you like to use iOS + Atlantis, and send the traffic to Proxyman for Windows?

Yeah, as I know our testers who has Windows can't see traffic in Proxyman, which we send through Atlantis

**Comment by radianttap** [comment]
> > Just wondering: Would you like to use iOS + Atlantis, and send the traffic to Proxyman for Windows?
> 
> Yeah, as I know our testers who has Windows can't see traffic in Proxyman, which we send through Atlantis

Same for me. My clients have multi-person QA team who are all using Windows and they would love the ability to see network traffic as they test the app functionality.

**Comment by radianttap** [comment]
Hello, is this feature planned for this year..?

**Comment by NghiaTranUIT** [comment]
We haven't started yet, since we are working on these high-priority features:
- Breakpoint (✅ Done)
- Map Local (✅ Done)
- Compose & Repeat (In progress)
- Diff (... Soon)

When these are done, I will revisit this ticket 👍
